### PR TITLE
Feat/561 review page styles

### PR DESCRIPTION
**Pull request by mlauren**
# Description
Includes review page styles by default

## Original issue(s)

department-of-veterans-affairs/va-forms-system-core#561

## Testing done
NA
## Screenshots
Screen Shot 2022-08-24 at 4 01 57 PM
Screen Shot 2022-08-25 at 9 43 27 AM

## Definition of done

- [ ] Package.json version has been updated to match Github release tag
- [x] Documentation has been updated, if applicable
- [x] A link to the original github issue has been provided
- [x] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/css/reviewPage.css
Comment (obliviga) [inline src/css/reviewPage.css:7]: Can use `.vads-u-text-align--right` potentially. Does it make sense to use these utility classes in this case, or does it not matter?

File: src/css/reviewPage.css
Comment (obliviga) [inline src/css/reviewPage.css:3]: Might be a good idea to use `.vads-u-color--primary-darker ` here, that way we can remove this entire CSS rule.

File: src/css/reviewPage.css
Comment (obliviga) [inline src/css/reviewPage.css:12]: `vads-u-color--gray`

File: src/css/reviewPage.css
Comment (obliviga) [inline src/css/reviewPage.css:16]: `vads-u-color--gray` and might be able to use `vads-u-font-size--md` if you could get away 17px instead of 18px: https://design.va.gov/foundation/utilities/font-size

File: src/css/reviewPage.css
Comment (obliviga) [inline src/css/reviewPage.css:8]: If you could get close to this value by using one of the utils [here](https://design.va.gov/foundation/utilities/margins), you might be able to remove this entire file!

**Comment by obliviga** [comment]
Just want to make sure -- would this type of change need a (fix) commit and package version update? @mlauren

File: src/css/reviewPage.css
Comment (mlauren) [reply src/css/reviewPage.css:8]: @obliviga rad, thanks for all the helpful comments!

**Comment by mlauren** [comment]
@obliviga I made some changes based on your recommendations. Once I get an approval I'll patch with (fix) commit and merge.

**Review by obliviga — APPROVED** [review]

**Pull request merged**
